### PR TITLE
Make contact us page furniture reusable

### PR DESCRIPTION
**Pull request by rBangay**
## What does this change?
The header and page container components in the contact us work are to be used as a template for future public facing routes on MMA. 

To accommodate this, the `ContactUsHeader` and `ContactUsPageContainer` components have been renamed to reflect there more general usage.

The left hand side of the new `SectionPageContainer` component is now narrower in keeping with the latest designs. 

There are a few more css amends to accommodate the new content widths.

## Images
Screenshot 2020-11-23 at 15 55 02

### CODE REVIEW

File: app/client/components/contactUs/contactUs.tsx [hunk 1]
```diff
@@ -159,10 +159,38 @@ export const ContactUs = (props: ContactUsProps) => {
     }
   };
 
+  const HeaderTitle = (
+    <span>
+      Need to contact us{" "}
```
Comment (kelvin-chappell) [inline app/client/components/contactUs/contactUs.tsx:164]: What does the `{" "}` do?

Comment (rBangay) [reply app/client/components/contactUs/contactUs.tsx:164]: Hey Kelvin ... it forces a space between `Need to contact us` and `about something?` on desktop and higher breakpoints (the "about something?" part is only visible for the larger breakpoints)

Comment (kelvin-chappell) [reply app/client/components/contactUs/contactUs.tsx:164]: I see. Thanks!

**Review by kelvin-chappell — APPROVED** [review]
Looks good to me! 👍

**Pull request merged**